src/leap/bitmask/backend/backend_proxy.py

```diff
@@ -36,33 +38,39 @@
 logger = logging.getLogger(__name__)
 
 
-class BackendProxy(object):
+class ZmqREQConnection(threading.Thread):
     """
-    The BackendProxy handles calls from the GUI and forwards (through ZMQ)
-    to the backend.
+    A threaded zmq req connection.
     """
 
-    PORT = '5556'
-    SERVER = "tcp://localhost:%s" % PORT
-
-    POLL_TIMEOUT = 4000  # ms
-    POLL_TRIES = 3
-
-    PING_INTERVAL = 2  # secs
-
-    def __init__(self):
-        generate_zmq_certificates_if_needed()
-
-        self._socket = None
+    def __init__(self, server_address, on_recv):
+        """
+        Initialize the connection.
 
-        self.settings = Settings()
```

Comment:
This line is needed here. See https://github.com/leapcode/bitmask_client/blob/develop/src/leap/bitmask/gui/mainwindow.py#L129